menu.py

```diff
@@ -25,61 +28,60 @@ class MainMenu(Menu):
     def __init__(self, game):
         Menu.__init__(self, game)
         self.state = "Start"
-        self.startx, self.starty = self.mid_w, self.mid_h + 30
+        self.startx, self.starty = self.mid_w, self.mid_h + 10
         self.optionsx, self.optionsy = self.mid_w, self.mid_h + 50
-        self.creditsx, self.creditsy = self.mid_w, self.mid_h + 70
+        self.creditsx, self.creditsy = self.mid_w, self.mid_h + 90
         self.cursor_rect.midtop = (self.startx + self.offset, self.starty)
 
     def display_menu(self):
         self.run_display = True
+        pygame.mixer.music.play(-1)
```

Comment:
Maybe add a comment:
(-1) to make the music repeat indefinitely

Reply:
I will add some comments bundle with my next pull request.